.vscode/settings.json

```diff
@@ -1,43 +1,45 @@
 {
-    "eslint.workingDirectories": [
-        {
-            "mode": "auto"
-        }
-    ],
-    "typescript.tsdk": "node_modules/typescript/lib",
+	"eslint.workingDirectories": [
+		{
+			"mode": "auto"
+		}
+	],
+	"typescript.tsdk": "node_modules/typescript/lib",
 
-    // Autodetecting tasks on a repo this size makes 'Tasks: Run Build Task' unusably slow.
-    // (See https://github.com/Microsoft/vscode/issues/34387)
-    "typescript.tsc.autoDetect": "off",
-    "npm.autoDetect": "off",
+	// Autodetecting tasks on a repo this size makes 'Tasks: Run Build Task' unusably slow.
+	// (See https://github.com/Microsoft/vscode/issues/34387)
+	"typescript.tsc.autoDetect": "off",
+	"npm.autoDetect": "off",
 
-    "files.associations": {
-        "tools/pipelines/*.yml": "azure-pipelines",
-        // good-fences' fence files support comments, so detect them as "jsonc" instead of "json":
-        "fence.json": "jsonc",
-        ".git-blame-ignore-revs": "ignore"
-    },
-    "[azure-pipelines].customSchemaFile": "",
-    "deno.enable": false,
-    "deno.lint": false,
-    "deno.unstable": false,
+	"files.associations": {
+		"tools/pipelines/*.yml": "azure-pipelines",
+		// good-fences' fence files support comments, so detect them as "jsonc" instead of "json":
+		"fence.json": "jsonc",
+		".git-blame-ignore-revs": "ignore"
+	},
+	"[azure-pipelines].customSchemaFile": "",
+	"deno.enable": false,
+	"deno.lint": false,
+	"deno.unstable": false,
 
-    // Custom dictionary for 'streetsidesoftware.code-spell-checker' extension.
-    "cSpell.words": [
-        "endregion",
-        "injective",
-        "losslessly",
-        "mitigations",
-        "multinomial",
-        "nonfinite",
-        "pseudorandomly",
-        "Tinylicious",
-        "tombstoned",
-        "TSDoc",
-        "unaugmented"
-    ],
+	// Custom dictionary for 'streetsidesoftware.code-spell-checker' extension.
+	"cSpell.words": [
+		"endregion",
+		"injective",
+		"losslessly",
+		"mitigations",
+		"multinomial",
+		"nacked",
+		"nonfinite",
+		"pseudorandomly",
+		"retriable",
```

Comment:
surveyed the code, retriable is more commonly used than retryable, so going with that, even typing this retryable gets re-underline via spell check, but retriable does not, so seems like the correct choice